src/ripple/app/main/Application.h

```diff
@@ -58,6 +58,8 @@ template <
     class Key,
     class T,
     bool IsKeyCache,
+    class SharedWeakUnionPointer,
```

Comment:
[nit] It makes sense to rename all usages of SharedWeakUnionPointer / SharedWeakUnionPointerType into SharedWeakCachePointer / SharedWeakCachePointerType for naming consistency.

Reply:
No support for this. Ignoring